### DISCUSSION TITLE
rust-course/3.5.1中关于内存泄漏的描述

### DISCUSSION THREAD

**Discussion by swellingWu** [sunface/rust-course#503] [内容建议]
在 main 函数结束前，a 和 b 的引用计数均是 2，随后 b 触发 Drop，此时引用计数会变为 1
应当是a和b的计数器都变为1吧，还是说我理解错了，这里写一个b会不会有些误导了（新手）

**Comment by sunface** [comment]
后面还有一句同理可得。
